### PR TITLE
Fix subscription term termUnit property

### PR DESCRIPTION
**Pull request by GGJason**
The TermUnit property is not in Subscription>Term. To fix this, I add a new enum property and share the TermUnit component.
image

### CODE REVIEW

**Review by santhoshb-msft — CHANGES_REQUESTED** [review]
Thank you for the changes, could we please fix the comments?

**Comment by JustSoftwareNPE** [comment]
@GGJason  Any update on this ? It would be great to get merged :)

File: Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json [hunk 1]
```diff
@@ -835,7 +835,7 @@
         "description": "Azure Active Directory OAuth2 Flow",
         "flows": {
           "clientCredentials": {
-            "tokenUrl": "https://login.microsoftonline.com/{your-app-tenant-id}/oauth2/token"
+            "tokenUrl": "https://login.microsoftonline.com/ce4d7f48-4562-42b4-9ca1-57e0e37438b9/oauth2/token"
```
[the remark this answers is not in the document]
Comment (GGJason) [reply Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json]: @santhoshb-msft  Sorry for late reply. I update the commit. Please merge the PR.

**Comment by santhoshb-msft** [comment]
@GGJason 
Autorest is throwing error, could you please check if its working for you when you run generate at https://github.com/microsoft/commercial-marketplace-client-dotnet
image

**Comment by GGJason** [comment]
@santhoshb-msft I found and solve the problem. Please review again. Thank you!

**Comment by GGJason** [comment]
> @GGJason Autorest is throwing error, could you please check if its working for you when you run generate at https://github.com/microsoft/commercial-marketplace-client-dotnet image

@santhoshb-msft I found and solve the problem. Please review again. Thank you!

**Comment by santhoshb-msft** [comment]
Fix already merged in #34 . Thank you!

**Pull request closed without merging**